tools/issue-labeler/Directory.Build.Props

```diff
@@ -0,0 +1,22 @@
+
+<Project>
+
+    <PropertyGroup>
+        <!-- Override output paths to use the default behavior -->
```

Comment:
We've got a fun case problem here - one file with `.Props` and one with `.props` - we should delete this copy.